static/js/components/EmploymentForm.js

```diff
@@ -100,7 +102,7 @@ class EmploymentForm extends ProfileFormFields {
     if ( ui.workHistoryEdit === true ) {
       let workHistoryRows = [];
       if ( !_.isUndefined(work_history) ) {
-        workHistoryRows = Object.entries(work_history).filter(([,entry]) =>
+        workHistoryRows = Object.entries(work_history).filter(([,entry]: WorkHistoryEntry[]) =>
```

Comment:
Can you change the type here to `[string, WorkHistoryEntry]` which is more accurate?